internal/server/server.go

```diff
@@ -196,3 +208,9 @@ func newInternalServer() (*http.Server, error) {
 
 	return &server, nil
 }
+
+// shouldEnableTrace - determines if opentracing should be enabled for the grpc method
+func shouldEnableTrace(ctx context.Context, fullMethodName string) bool {
```

Comment:
nit: Actually, the function determines which path to exclude from the trace. So rename to shouldDisableTrace or shouldSkipTrace

Reply:
Check the implementation of `UnaryServerInterceptor`
The filter method needs to return `false` to skip tracing
Hence.. Should enable trace? `false` = no